netbout-web/pom.xml

```diff
@@ -326,13 +326,15 @@
                             <npmVersion>2.11.3</npmVersion>
                         </configuration>
                     </execution>
+                    <!-- @todo #914 investigate error messages of the following
```

Comment:
@jhyle and I believe Travis should warn here about missing time... Maybe you need to put `@todo` to a new line?